portfolio/src/main/webapp/script-home.js

```diff
@@ -139,8 +144,145 @@ function deleteComments() {
  * Creates a map using the Google Maps API.
  */
 function initMap() {
-  const map = new google.maps.Map(document.getElementById('map'), {
+  // A custom color scheme of the map + basic properties for initialization
+  // (like initial center of map + zoom level)
+  const mapOptions = {
```

Comment:
mapOptions and mapLabel seem to be non-changing data. Make them global constants